remote/gitea/parse.go

```diff
@@ -0,0 +1,107 @@
+package gitea
+
+import (
+	"io"
+	"net/http"
+
+	"github.com/drone/drone/model"
+)
+
+const (
+	hookEvent       = "X-Gogs-Event"
```

Comment:
You are right, I thought we already added Gitea there: https://github.com/go-gitea/gitea/blob/master/models/webhook.go#L539-L542